app/react/I18N/components/EditTranslationForm.js

```diff
@@ -133,12 +132,22 @@ export class EditTranslationForm extends Component {
             </ul>
           </div>
           <div className="settings-footer">
-            <BackButton to="/settings/translations" />
-            {importButton}
-            <button type="submit" className="btn btn-success save-template">
-              <Icon icon="save" />
-              <span className="btn-label">{t('System', 'Save')}</span>
-            </button>
+            <div className="btn-cluster">
+              <BackButton to="/settings/translations" className="btn-plain" />
+              {importButton}
+            </div>
+            <div className="btn-cluster content-right">
+              <I18NLink to="/settings/translations" className="btn btn-extra-padding btn-default">
+                <span className="btn-label">
+                  <Translate>Cancel</Translate>
```

Comment:
Same as above, I don't recall if these translate components will work with existing translations withouth the prop `translationKey`.